src/Filter/Factory/ToInternationalPhoneNumberFactory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\I18n\PhoneNumber\Filter\Factory;
+
+use Laminas\I18n\PhoneNumber\Factory\ConfigurationTrait;
+use Laminas\I18n\PhoneNumber\Filter\ToInternationalPhoneNumber;
+use Psr\Container\ContainerInterface;
+
+final class ToInternationalPhoneNumberFactory
+{
+    use ConfigurationTrait;
```

Comment:
Discussed above: let's avoid using the trait, and rather use static methods instead